src/Common/src/System/IO/PathInternal.Windows.cs

```diff
@@ -81,13 +81,25 @@ internal static bool IsValidDriveChar(char value)
             return ((value >= 'A' && value <= 'Z') || (value >= 'a' && value <= 'z'));
         }
 
+        private static bool EndsWithPeriodOrSpace(string path)
+        {
+            if (string.IsNullOrEmpty(path))
```

Comment:
>string.IsNullOrEmpty(path) [](start = 16, length = 26)

This allows a completely blank file name and them the user can create " " and "    " hopefully nobody will do that, but yes, Windows does support it :smile:

Reply:
Yeah, not recommended. We still make it difficult to create them by accident, but at least now we won't fall over.